vocab_files/observable_property_concepts/in-canopy-sky.ttl

```diff
@@ -6,13 +7,17 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/test/dawe-cv/85bdd25a-fa08-49de-9f0b-98895cb79aa6>
     a skos:Concept ;
+    dcterms:source "Laws M, McCallum K, Bignall J, O’Neill S, Sparrow B. (unpublished draft) ‘Fire module’ in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:altLabel "in canopy sky (ics) status" ;
```

Comment:
I would have the pref label as `in-canopy sky` or `in-canopy sky status` and the alt label as `ICS`